components/Seo.php

```diff
@@ -53,7 +56,7 @@ public function onRender()
 
         if ($isBlogPage !== false) {
             $this->getDataFromBlogPost();
-        } else if ($isStaticPage !== false) {
+        } elseif ($isStaticPage !== false) {
```

Comment:
I prefer to have `else if` than `elseif`

Reply:
done